SU2_CFD/src/output/COutput.cpp

```diff
@@ -1916,7 +1928,7 @@ void COutput::Postprocess_HistoryData(CConfig *config){
     if (currentField.fieldType == HistoryFieldType::COEFFICIENT){
       if (config->GetTime_Domain()){
         windowedTimeAverages[historyOutput_List[iField]].addValue(currentField.value,config->GetTimeIter(), config->GetStartWindowIteration()); //Collecting Values for Windowing
```

Comment:
right now this was creating a default average object (square window), because operator [] of a map inserts if something does not exist yet.
```suggestion
      if (config->GetTime_Domain()){
        auto& timeAverage = windowedTimeAverages.at(historyOutput_List[iField]);
        timeAverage.addValue(currentField.value,config->GetTimeIter(), config->GetStartWindowIteration()); //Collecting Values for Windowing
```

Reply:
I'm pretty sure I had this somewhere in the COutput constructor, but the change got somehow lost... my bad!